code/src/components/HappyThoughts.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import moment from 'moment'
+import './happyThoughts.css'
+
+export const HappyThoughts = props => {
+    console.log("props", props)
+    const { message, hearts, createdAt, _id } = props.thought
+
+        const handleClick = () => {
+            fetch(`https://technigo-thoughts.herokuapp.com/${_id}/like`, {
+            method: "POST",
+            body: "",
+            headers: { "Content-Type": "application/json" }
+            }).then(() => props.onLiked(_id))
+        }
+
+    return (
+        <article>
+            <h3>{message}</h3>
+            <div className="heart-time">
+                <p>
+                <button onClick={handleClick}>
```

Comment:
Here you can add a `className` to your button that will be activated when clicking on it. Example:
```
className={`heart ${clicked && " heart-filled"}`}
```